applications/asset_tracker/src/light_sensor/light_sensor.h

```diff
@@ -49,6 +49,21 @@ int light_sensor_init_and_start(const light_sensor_data_ready_cb cb);
  */
 int light_sensor_get_data(struct light_sensor_data *const data);
 
+/**
+ * @brief Set light sensor's poll/send interval.
+ *
+ * @param interval_s Interval, in seconds. 0 to disable.
+ *
+ */
+void light_sensor_set_send_interval(const s32_t interval_s);
```

Comment:
I'd consider using `u32_t` here, we don't support negative intervals.

Reply:
i kept it signed since that is what what the delayed work function expects.
i'll update.